tests/Oryx.Integration.Tests/NodeEndToEndTests.cs

```diff
@@ -13,137 +13,28 @@
 
 namespace Microsoft.Oryx.Integration.Tests
 {
-    [Trait("category", "node")]
-    public class NodeEndToEndTests : PlatformEndToEndTestsBase
+    public class NodeEndToEndTestsBase : PlatformEndToEndTestsBase
```

Comment:
BTW do you plan to even do this similar thing for Node tests present in `Oryx.RuntimeImage.Tests`?

Reply:
yes